crates/shadowsocks/src/net/sys/unix/bsd/freebsd.rs

```diff
@@ -251,8 +251,10 @@ pub async fn bind_outbound_udp_socket(bind_addr: &SocketAddr, _config: &ConnectO
         UdpSocket::from_std(socket.into())?
     };
 
-    if let Err(err) = set_disable_ip_fragmentation(af, &socket) {
-        warn!("failed to disable IP fragmentation, error: {}", err);
+    if ! config.udp_allow_fragmentation {
```

Comment:
Emm, the space after `!` is not needed. You should run `cargo fmt` before commit.